src/schema/mbs_types.ts

```diff
@@ -0,0 +1,267 @@
+/*
+ * This file is part of ciboard-server
+ *
+ * Copyright (c) 2021 Andrei Stepanov <astepano@redhat.com>
+ * Copyright (c) 2022 Matěj Grabovský <mgrabovs@redhat.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import _ from 'lodash';
+import debug from 'debug';
+import * as graphql from 'graphql';
+import {
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { delegateToSchema } from '@graphql-tools/delegate';
+
+import schema from './schema';
+import { CommitObject, DistGitInstanceInputType } from './distgit_types';
+import {
+  KojiBuildTagsType,
+  KojiHistoryType,
+  KojiInstanceInputType,
+} from './koji_types';
+
+const log = debug('osci:mbs_types');
+
+export interface MbsTaskFields {
+  component: string;
+  id?: number;
```

Comment:
1. In the source code bellow you use also `id` for MBS ID. This id is task-id.  Maybe add taskId? mbsId? It would greatly help in code-reading.
2. Why `id` is optional? Task itself imposes it has id.

Reply:
1 - true.
2 - please add this as a comment near this code.